Services/WindowServer/ClientConnection.h

```diff
@@ -125,6 +125,8 @@ class ClientConnection final
     virtual OwnPtr<Messages::WindowServer::SetWindowBaseSizeAndSizeIncrementResponse> handle(const Messages::WindowServer::SetWindowBaseSizeAndSizeIncrement&) override;
     virtual void handle(const Messages::WindowServer::EnableDisplayLink&) override;
     virtual void handle(const Messages::WindowServer::DisableDisplayLink&) override;
+    virtual OwnPtr<Messages::WindowServer::GetClipboardContentsResponse> handle(const Messages::WindowServer::GetClipboardContents &) override;
+    virtual OwnPtr<Messages::WindowServer::SetClipboardContentsResponse> handle(const Messages::WindowServer::SetClipboardContents &) override;
```

Comment:
These API's are no longer in `Messages::WindowServer`. You need to remove generated files from your source tree after the cmake changes. Something like `git clean -qdfx` should do the trick.

Reply:
I see, got it, I will do so, thanks!